ldm/simplet2i.py

```diff
@@ -637,3 +641,27 @@ def _run_gfpgan(self, image, strength):
             res = Image.blend(image, res, strength)
 
         return res
+
+    # shows how the prompt is tokenized 
+    # usually tokens have '</w>' to indicate end-of-word, 
+    # but for readability it has been replaced with ' '
+    def _log_tokenization(self, text):
+        if not self.log_tokenization:
+            return
+        tokens = self.model.cond_stage_model.tokenizer._tokenize(text)
+        tokenized = ""
+        discarded = ""
+        usedTokens = 0
+        totalTokens = len(tokens)
+        for i in range(0,totalTokens):                
```

Comment:
Can't this simply be `for token in tokens:`, and then actually **remove** the `cond_stage_model.max_length` check below, given that you'll only get as many tokens from the iterator as actually exist...